### PR TITLE
Update launch

### PR DESCRIPTION
**Pull request by KmakD**
Ideally we'll use WH01.yaml, WH02.yaml, WH04.yaml from panther_description but for now I put them into panther_bringup

### CODE REVIEW

File: panther_bringup/config/WH01.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+wheel_radius: 0.1825
```
Comment (Kotochleb) [inline panther_bringup/config/WH01.yaml]: I would use those parameters from panther_description

File: panther_bringup/config/WH02.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+wheel_radius: 0.1016
```
Comment (Kotochleb) [inline panther_bringup/config/WH02.yaml]: I would use those parameters from panther_description

File: panther_bringup/config/WH04.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+wheel_radius: 0.1016
```
Comment (Kotochleb) [inline panther_bringup/config/WH04.yaml]: I would use those parameters from panther_description

File: panther_bringup/launch/bringup.launch [hunk 4]
```diff
@@ -1,19 +1,29 @@
 <launch>
+  <arg name="panther_version" value="$(optenv PANTHER_ROBOT_VERSION 1.0)" />
+  <arg name="wheel_type" default="WH01" doc="Type of wheel. Possible: 'WH01', 'WH02', 'WH04', 'custom'" />
+
+  <arg name="wheel_config_file" if="$(eval wheel_type == 'custom')" />
+  <arg name="wheel_config_file" value="$(find panther_bringup)/config/WH01.yaml" if="$(eval wheel_type == 'WH01')" />
+  <arg name="wheel_config_file" value="$(find panther_bringup)/config/WH02.yaml" if="$(eval wheel_type == 'WH02')" />
+  <arg name="wheel_config_file" value="$(find panther_bringup)/config/WH04.yaml" if="$(eval wheel_type == 'WH04')" />
```
Comment (Kotochleb) [inline panther_bringup/launch/bringup.launch]:
```suggestion
  <arg name="wheel_config_file" value="$(find panther_description)/config/WH01.yaml" if="$(eval wheel_type == 'WH01')" />
  <arg name="wheel_config_file" value="$(find panther_description)/config/WH02.yaml" if="$(eval wheel_type == 'WH02')" />
  <arg name="wheel_config_file" value="$(find panther_description)/config/WH04.yaml" if="$(eval wheel_type == 'WH04')" />
```

**Review by Kotochleb — CHANGES_REQUESTED** [review]

File: panther_bringup/config/WH01.yaml (hunk 1 above)
Comment (KmakD) [reply panther_bringup/config/WH01.yaml]: As I stated in PR description, this is temporary. Panther_description for ros1 is still waiting for changes to be merged. This apply to other comments

File: panther_bringup/launch/bringup.launch (hunk 4 above)
Comment (pkowalsk1) [reply panther_bringup/launch/bringup.launch:8]: +1

Comment (pkowalsk1) [reply panther_bringup/launch/bringup.launch]: +1

**Review by pkowalsk1 — CHANGES_REQUESTED** [review]
In my opinion, we can apply the changes @Kotochleb mentioned regarding WH0*.yaml files.

**Review by Kotochleb — APPROVED** [review]

**Review by pkowalsk1 — APPROVED** [review]

**Pull request merged**